staging/src/k8s.io/code-generator/cmd/conversion-gen/main.go

```diff
@@ -14,20 +14,38 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// conversion-gen is a tool for auto-generating Conversion functions.
+// conversion-gen is a tool for auto-generating functions that convert
+// between internal and external types.  A general conversion task
+// involves three packages: the one containing the internal types, the
+// one containing the external types, and the destination (i.e., the
+// one containing the conversion functions).
 //
-// Given a list of input directories, it will scan for "peer" packages and
-// generate functions that efficiently convert between same-name types in each
-// package.  For any pair of types that has a
-//     `Convert_<pkg1>_<type>_To_<pkg2>_<Type()`
-// function (and its reciprocal), it will simply call that.  use standard value
-// assignment whenever possible.  The resulting file will be stored in the same
-// directory as the processed source package.
+// For each conversion task, `conversion-gen` will generate functions
+// that efficiently convert between same-name types in each of the two
+// packages.  The generated functions include ones named
+//     autoConvert_<pkg1>_<type>_To_<pkg2>_<type>
+// for each such ordered pair of types.  If the destination package
+// does not contain one in a non-generated file, a function named
+//     Convert_<pkg1>_<type>_To_<pkg2>_<Type()
+// is also generated and it simply calls the `autoConvert...`
+// function.  The generated conversion functions use standard value
+// assignment wherever possible.  For compound types, the generated
+// conversion functions call the `Convert...` functions for the
+// subsidiary types.  Thus developers can override the behavior for
+// selected types.
 //
-// Generation is governed by comment tags in the source.  Any package may
-// request Conversion generation by including a comment in the file-comments of
-// one file, of the form:
-//   // +k8s:conversion-gen=<import-path-of-peer-package>
+// Given a list of input directories, `conversion-gen` will scan for
```

Comment:
Is the "list of input directories" the value specified via the "--input-dirs" flag? If so, can we explicitly say so? 
Something like "To start code generation, invoke the `conversion-gen` binary with flag `--input-dirs=<input directories>`. `conversion-gen` will scan for ..."

Reply:
Updated.